package.json

```diff
@@ -44,20 +44,23 @@
     "apollo-upload-client": "11.0.0",
     "cordova-android": "8.0.0",
     "cordova-ios": "5.0.1",
+    "cordova-plugin-aerogear-metrics": "npm:@aerogear/cordova-plugin-aerogear-metrics",
+    "cordova-plugin-app-version": "0.1.9",
     "cordova-plugin-badge": "0.8.8",
     "cordova-plugin-device": "2.0.3",
     "cordova-plugin-dialogs": "2.0.2",
     "cordova-plugin-inappbrowser": "3.1.0",
     "cordova-plugin-ionic-keyboard": "2.1.3",
     "cordova-plugin-ionic-webview": "4.1.1",
+    "cordova-plugin-iroot": "npm:@aerogear/cordova-plugin-iroot",
```

Comment:
nice trick!

Reply:
The only bad thing is that cordova force me to do also this: `"@aerogear/cordova-plugin-iroot": "npm:@aerogear/cordova-plugin-iroot@0.8.1"`